tests/scenarios/test_functionality_nimbus.py

```diff
@@ -126,3 +137,90 @@ def check_new_tab(selenium):
     # get the last tab
     selenium.switch_to.window(selenium.window_handles[-1])
     assert "about:newtab" in selenium.current_url
+
+
+@then("The user will install a language pack")
+def install_acholi_language_pack(selenium, request):
+
+    add_button_locator = (By.CSS_SELECTOR, "#add")
+    addon_installed_locator = (By.CSS_SELECTOR, "#appMenu-addon-installed-notification")
+    add_to_firefox_locator = (
+        By.CSS_SELECTOR,
+        "#addon-install-confirmation-notification .popup-notification-primary-button",
+    )
+    browser_dialog_box_locator = (By.CSS_SELECTOR, "#BrowserLanguagesDialog")
+    root_dialog_box_locator = (
+        By.CSS_SELECTOR,
+        "vbox.dialogOverlay:nth-child(1) > vbox:nth-child(1) > browser:nth-child(2)",
```

Comment:
`nth-child()` always scares me. I guess its technically a _bit_ safer since you're adding specificity via `vbox:` and `browser:` prefixes. Too bad there aren't better class names or IDs/attributes we could target. 🤷

Reply:
They reuse the dialog boxes on the preferences page, so this is a bit hard to dial in